.github/workflows/release.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
         GOOGLE_APPLICATION_CREDENTIALS: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
         GOOGLE_PROJECT: ${{ secrets.GOOGLE_PROJECT }}
         GOOGLE_LOCATION: ${{ secrets.GOOGLE_LOCATION }}
+        KONG_ENTERPRISE_LICENSE: ${{ secrets.KONG_ENTERPRISE_LICENSE }}
```

Comment:
I expect this one to work, as it appears you've added the `KONG_ENTERPRISE_LICENSE` to the `gcloud` environment :+1: